### ISSUE TITLE
After install this app, display error when i first open this.

### ISSUE THREAD

**Issue by BackTrackCRoot** [lhc70000/iina#1124]
```
Process:               IINA [766]
Path:                  /Applications/IINA.app/Contents/MacOS/IINA
Identifier:            com.colliderli.iina
Version:               ???
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           IINA [766]
User ID:               501

Date/Time:             2017-10-18 11:13:27.393 +0800
OS Version:            Mac OS X 10.10.5 (14F2511)
Report Version:        11

Time Awake Since Boot: 3700 seconds
Time Since Wake:       1000 seconds

Crashed Thread:        0

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000002, 0x0000000000000000

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: @rpath/libmpv.1.dylib
  Referenced from: /Applications/IINA.app/Contents/MacOS/IINA
  Reason: no suitable image found.  Did find:
	/Applications/IINA.app/Contents/MacOS/../Frameworks/libmpv.1.dylib: code signature invalid for '/Applications/IINA.app/Contents/MacOS/../Frameworks/libmpv.1.dylib'

	/Applications/IINA.app/Contents/MacOS/../Frameworks/libmpv.1.dylib: code signature invalid for '/Applications/IINA.app/Contents/MacOS/../Frameworks/libmpv.1.dylib'

	/Applications/IINA.app/Contents/MacOS/../Frameworks/libmpv.1.dylib: code signature invalid for '/Applications/IINA.app/Contents/MacOS/../Frameworks/libmpv.1.dylib'

Binary Images:
    0x7fff6b80e000 -     0x7fff6b84492f  dyld (353.2.4) <3C068A0D-69D5-3AD6-86D9-6261D3B1564E> /usr/lib/dyld
```

**Comment by lhc70000** [comment]
If you're using 0.0.14, please switch back to [0.0.13](https://github.com/lhc70000/iina/releases/tag/v0.0.13).

**Comment by BackTrackCRoot** [comment]
Thanks~ i can open ~

**Comment by BackTrackCRoot** [comment]
But when i click open url , it nothing to do ...见鬼。

**Comment by lhc70000** [comment]
Fixed in 0.0.14.1.
